techniques/system/common/1.0/promises.st

```diff
@@ -146,12 +146,13 @@ bundle common va
       "ncf_inputs"            slist => filter("${ncf_path}/cf_null", "raw_ncf_inputs", "false", "true", 10000);
 
     # create the final input list after ncf
-    # all other inputs are loaded by body file control in rudder-system-directives.cf and rudder-directives.cf
+    # all other inputs are loaded by body file control in rudder-system-directives.cf
       "inputs_list" slist => { @{ncf_inputs} };
 
 &if(INITIAL)&
+      # rudder-directives.cf is needed, even if empty since we call it in the bundle sequence
       # Idealy these should be generated at compile time from metadata.xml and put into variable.json
-      "common_input_list" slist => { "common/1.0/common.cf", "common/1.0/cf-serverd.cf", "common/1.0/rudder-groups.cf", "common/1.0/hooks.cf", "common/1.0/cron-setup.cf", "common/1.0/site.cf", "common/1.0/update.cf", "common/1.0/monitoring.cf", "common/1.0/restart-services.cf", "common/1.0/internal-security.cf",  "common/1.0/environment-variables.cf", "common/1.0/properties.cf" };
+      "common_input_list" slist => { "rudder-directives.cf", "common/1.0/common.cf", "common/1.0/cf-serverd.cf", "common/1.0/rudder-groups.cf", "common/1.0/hooks.cf", "common/1.0/cron-setup.cf", "common/1.0/site.cf", "common/1.0/update.cf", "common/1.0/monitoring.cf", "common/1.0/restart-services.cf", "common/1.0/internal-security.cf",  "common/1.0/environment-variables.cf", "common/1.0/properties.cf" };
```

Comment:
this file is not part of the common technique, I is probably not the right place

Reply:
it is